build_projects/dotnet-cli-build/build.sh

```diff
@@ -101,6 +101,15 @@ then
     ulimit -n 1024
 fi
 
+# Temporary logic to turn core dumps on so we can catch exit code 138 errors on OSX
+if [ "$(uname -s)" == "Darwin" ]; then
+  # If there are no core files already in /cores/, we'll enable core dump
+  # generation for this shell.
+  if [ "$(ls -A /cores)" ]; then 
```

Comment:
> Please do the reverse (i.e. disable the generation) once the build script ends

Is this necessary?  @adityamandaleeka didn't do the reverse in the buildtools change.

> and do a "ls -l /cores" as well.

Can you explain a little more what you want here?  What do you want to see happening?